modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -618,6 +618,11 @@ export class Wallet implements IWallet {
       .send(filteredParams)
       .result();
 
+    response.txInfo?.unspents?.map((u) => {
```

Comment:
`map` method doesn't change the `value` to `bigint` in the `response.txInfo.unspents` array right? It returns new `bigint` array.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map